### DISCUSSION TITLE
Metadata API end points

### DISCUSSION THREAD

**Discussion by alon710** [supabase/supabase#2333] [Questions] [answered]
Hello, I'm trying to add users to Supabse studio, now I'm working on the `policies` page and I don't understand where can I get the data from

I have the PostREST API URL but I don't have access to this data from this interface

Can someone please guide me to all Supabase APIs documentation?

Thanks!

**Comment by steve-chavez** [comment, marked as answer]
Hey @alon710,

> now I'm working on the policies page and I don't understand where can I get the data from

If I understand you correctly, do you want RLS policies metadata from postgres?

If so, getting that info is not done through PostgREST, but through [postgrest-meta](https://github.com/supabase/postgres-meta).

**Reply by alon710** [reply]
@steve-chavez I think that is what  I was looking for, where can I find the full documentation or usage examples?

Thanks!

**Reply by steve-chavez** [reply]
>  I think that is what I was looking for, where can I find the full documentation or usage examples?

IIRC, it outputs OpenAPI, so this could help with some usage examples. There's no docs for now AFAICT, maybe raise an [issue](https://github.com/supabase/postgres-meta/issues) on the repo for further guidance.
